src/index.tsx

```diff
@@ -1,7 +1,10 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { SimpleDetailsList } from './SimpleDetailsList';
+import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
 
 ReactDOM.render(
-  <SimpleDetailsList />, 
+  <Fabric>
+    <SimpleDetailsList />
+  </Fabric>,
```

Comment:
Adding the Fabric wrapper around the application will add some root classes to conditionally show focus rectangle if the user uses the keyboard.